core-bundle/src/Resources/contao/dca/tl_files.php

```diff
@@ -210,29 +210,29 @@
 		(
 			'label'                   => &$GLOBALS['TL_LANG']['tl_files']['importantPartX'],
 			'inputType'               => 'text',
-			'eval'                    => array('rgxp'=>'natural', 'nospace'=>true, 'tl_class'=>'w50 clr'),
-			'sql'                     => "int(10) NOT NULL default 0"
+			'eval'                    => array('rgxp'=>'digit', 'nospace'=>true, 'tl_class'=>'w50 clr'),
```

Comment:
Can the important part now be negative?

Reply:
No.

Reply:
But `digit` allows for negative numbers whereas `natural` does not. But `natural` does not allow for decimal points, I guess that's why you changed it? Btw. `nospace` is not really required as digit/natural rgxp does not allow for whitespaces anyway.

Reply:
> But `natural` does not allow for decimal points, I guess that's why you changed it?

Yes